yarn.lock

```diff
@@ -13048,6 +13048,28 @@ __metadata:
   languageName: node
   linkType: hard
 
+"yjs-widgets@npm:^0.4":
```

Comment:
Mmh it looks like there are two yjs-widgets in the `yarn.lock` now. `0.4.0` and `0.3.8`.

Reply:
It's usually not a good idea to modify the `yarn.lock` manually, what I usually do is read the `yarn.lock` to understand who is pulling the old yjs-widgets version.

And it seems like it's jupytercad-app that pulls it: https://github.com/jupytercad/JupyterCAD/blob/1474ba8d33132a0d8fedda961ef3808cc4a0768e/yarn.lock#L873

So you need to modify the `package.json` of jupytercad-app with the new pinning, then re-run `jlpm install` to update the `yarn.lock`

Reply:
Done.